lib/hyrax/specs/shared_specs/hydra_works.rb

```diff
@@ -76,3 +76,54 @@
     end
   end
 end
+
+RSpec.shared_examples 'a Hyrax::PcdmFileSet' do
+  subject(:fileset)   { described_class.new }
+  let(:adapter)       { Valkyrie::MetadataAdapter.find(:test_adapter) }
+  let(:persister)     { adapter.persister }
+  let(:query_service) { adapter.query_service }
+
+  it_behaves_like 'a Hyrax::Resource'
+  it_behaves_like 'a model with core metadata'
+
+  it { is_expected.not_to be_collection }
+  it { is_expected.not_to be_file }
+  it { is_expected.to be_file_set }
+  it { is_expected.to be_pcdm_object }
+  it { is_expected.not_to be_work }
+
+  describe 'files' do
```

Comment:
Does this need tests for the other attributes (`original_file_id`, etc...)? 

Or we could hold off on adding those attributes until something is going to use them?

Ideas?

Reply:
I added them in the second commit.